### PR TITLE
Update deploy-on-aws-eks.md

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Review by lilin90 — APPROVED** [review]
LGTM

File: zh/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-on-aws-eks/']
...
-    - TiDB 所在节点: `c5.4xlarge`
-    - TiKV 或 TiFlash 所在节点: `m5.4xlarge`
-- 推荐存储：因为 AWS 目前已经支持 [EBS gp3](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html#gp3-ebs-volume-type) 卷类型，建议使用 EBS gp3 卷类型。对于 gp3 配置，推荐:
-    - TiKV: 400 MiB/s 与 4000 IOPS
-    - TiFlash: 625 MiB/s 与 6000 IOPS
+- 推荐机型：出于性能考虑，推荐：
+    - PD 所在节点：`c5.xlarge`
+    - TiDB 所在节点：`c5.4xlarge`
+    - TiKV 或 TiFlash 所在节点：`m5.4xlarge`
+- 推荐存储：因为 AWS 目前已经支持 [EBS gp3](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html#gp3-ebs-volume-type) 卷类型，建议使用 EBS gp3 卷类型。对于 gp3 配置，推荐：
```
Comment (KanShiori) [inline zh/deploy-on-aws-eks.md:39]:
```suggestion
- 推荐存储：因为 AWS 目前已经支持 [EBS gp3](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/general-purpose.html#gp3-ebs-volume-type) 卷类型，建议使用 EBS gp3 卷类型。对于 gp3 配置，推荐：
```

**Review by lilin90 — COMMENTED** [review]
The new link LGTM

**Pull request merged**